lib/ace/ext/papeeria/katex-demo.coffee

```diff
@@ -0,0 +1,31 @@
+define (require, exports, module) ->
```

Comment:
Here I'm not calling `define` with arguments `require, exports, module`, I'm calling `define` with 1 argument, which is a function `(require, exports, module) -> <function body>`. And I'm using CoffeeScript feature, where braces around arguments are optional. So, with all the braces, it would look like this:

    define((require, exports, module) -> ...)

So, combined with your other comments, I guess I won't use this feature.